m-queue.js

```diff
@@ -56,8 +59,9 @@ module.exports = function(RED) {
         var storeName = node.storeName
         var newValue = node.newValue
         var keepNewestDefault = node.keepNewestDefault
-        var maxSizeDefault = node.maxSizeDefault
+        var maxSizeDefault = node.maxSizeDefault < 0 ? Infinity : node.maxSizeDefault
```

Comment:
@drmibell This was a bug because all your queues were created at a negative size if it was set by default. It was correct if they were updated using maximumCmd